src/main/java/org/opentripplanner/standalone/CommandLineParameters.java

```diff
@@ -70,6 +69,15 @@
             description = "Pass the graph to the server in-memory after building it, and saving to disk.")
     public boolean preFlight;
 
+    @Parameter(names = { "--version", },
+            description = "Print the version, and then exit.")
+    public boolean version = false;
+
+    @Parameter(names = { "--version-fragment" },
+            description = "Print the a fragment of the version, and then exit. Use one of [`version`, `major`, `minor`, `incremental`, " +
+                    "`qualifier`, `commit` ] to specify which fragment")
+    public String versionFragment = DEFAULT_VERSION_FRAGMENT;
```

Comment:
_need_ would indeed be a stretch. It makes things a bit easier over doing somethimg like 
```
java -jar otp.jar --version | grep"commit" | cut -c8-
```
Which isn't that hard to do. Especially because I coudn't combine the two I see that it might be overkill.